pkgs/stdenv/freebsd/autotools-abspath.sh

```diff
@@ -0,0 +1,11 @@
+postPatchHooks+=(autotoolsAbspathHook)
+
+autotoolsAbspathHook() {
+    if [ -z "$dontAutopatchAutotools" ]; then
+        AUTOTOOLS_FILES="configure config.guess"
+        for filename in $AUTOTOOLS_FILES; do
+            find . -name $filename -exec \
+                sed -e 's@/usr/bin/uname@uname@g' -i '{}' ';'
+        done
```

Comment:
I guess this list doesn't get much longer? And we should only sed files.

```suggestion
        find . \( -name 'configure' -o -name 'config.guess' \) -type f -exec \
            sed -e 's@/usr/bin/uname@uname@g' -i '{}' ';'
```